configuration_script.sh

```diff
@@ -0,0 +1,50 @@
+setup_source_cluster() {
+    #This function creates a new project and application for migration on source cluster.
+    oc login -u opentlc-mgr -p 'r3dh4t1!' https://master.rdj2ocp3.mg.dog8code.com:443 --insecure-skip-tls-verify
+    if (oc get project nandini 2>/dev/null); then
+        oc delete project nandini
+    fi
+    sleep 30
+    oc new-project nandini
+    oc new-app django-psql-persistent
+    curl $(oc get routes -n nandini | grep django| awk '{print $2}')
+}
+
+setup_target_cluster() {
+    #This function deletes any existing plan(s) and target namespace on the target cluster.
+    #TO DO: Remove hardcoding of project name.
+    oc login https://api.cluster-rdj2ocp4b.rdj2ocp4b.mg.dog8code.com:6443 -u kubeadmin -p  8IuXo-ChDKW-cKxkG-X3ML3 --insecure-skip-tls-verify
```

Comment:
@nachandr , This should be sanitized.

```suggestion
    oc login https://api.cluster-your-ocp4-cluster.your-ocp4-cluster.mg.dog8code.com:6443 -u USERNAME -p  PASSWORD --insecure-skip-tls-verify
```